<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectionsWindow</class>
 <widget class="QMainWindow" name="CollectionsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskManager | Collections</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="collectionMainLayout">
     <item>
      <widget class="QPushButton" name="collectionBackButton">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Log Out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="collectionLineEdit">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Collection name</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="collectionSubLayout">
       <item>
        <widget class="QPushButton" name="collectionAddButton">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Add collection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="collectionViewButton">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>View Collection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="collectionDeleteButton">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Delete Collection</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="collectionList">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
